Header header
float64 Img_local_X
float64 Img_local_Y
float64 width
float64 height
sensor_msgs/Image image_roi
uint8 flag_roi

